content/news/2020-12-01-about-me.md

```diff
@@ -0,0 +1,74 @@
+title: All about my Outreachy internship with Mixxx DJ: Intro
+authors: Aanyu Deborah Oduman
+tags: website, Outreachy, internhip
+date: 2020-12-01 23:15:00
+## A little bit about me.
```

Comment:
Please leave a blank line between the metadata and the content. Also, let's add a little introduction so that readers know what this is about.

```suggestion
date: 2020-12-01 23:15:00

*In October, [we announced[({filename}/news/2020-10-01-outreachy.md) that Mixxx has been accepted into the [Outreachy program](https://www.outreachy.org/), which provides three-month internships to work in Free and Open Source Software (FOSS). We received many promising applications, and picked Deborah Aanyu as our first contributor. She will mainly work on improving the Mixxx manual and video tutorials and document her work in a series of posts here on our website.*

## A little bit about me.
```

Reply:
Ping @ywwg please let me know if this is okay.

Reply:
As written, this creates more separation between mixxx and deborah than I like.  I think this would work better if, @deborahtrez, you could write this from your own perspective -- "In October, Mixxx announced that it had been accepted into the Outreachy Organization. [link to other posts]...  I was selected as the first contributor!"...

Reply:
Yes, true. That would be even better. @deborahtrez can you write a little introduction?